contents/blog/experiments.md

```diff
@@ -0,0 +1,106 @@
+---
+title: What launching Experimentation taught us about running effective A/B tests
```

Comment:
On title, I'm imagining something like this:

Introducing Experimentation (and what it taught us about running effective A/B tests)
Introducing Experimentation and x things we learnt about running effective A/B tests

Reply:
Makes sense.

Introducing Experimentation and 3 things we learnt about running effective A/B tests

Reply:
Going to stick with your title